packages/browser-extension/src/scripts/dialog.js

```diff
@@ -90,7 +92,18 @@ const reportDialogHtml = `
               tabindex="0">
               ${browser.i18n.getMessage('reportDialog_popupShowUpOption')}
             </report-dialog-radio>
+            <report-dialog-radio 
+            aria-checked="false" 
+            data-value="6" 
+            role="radio" 
+            tabindex="0">
+            ${browser.i18n.getMessage('reportDialog_ScrollingIssues')}
+          </report-dialog-radio>
+          <textarea-report-dialog>
```

Comment:
This component is not needed here. Please remove it, leave only the text area.

Reply:
Done